xbmc/network/WebServer.cpp

```diff
@@ -597,7 +597,9 @@ bool CWebServer::ProcessPostData(const HTTPRequest& request,
     if (!postDataHandled)
     {
       m_logger->error("failed to handle HTTP POST data for {}", request.pathUrl);
-#if (MHD_VERSION >= 0x00095213)
+#if (MHD_VERSION >= 0x00097400)
+      connectionHandler->errorStatus = MHD_HTTP_CONTENT_TOO_LARGE;
```

Comment:
no, only since 0.9.74. Snippet of the warning:

```
[build] [6/13  15% :: 0.028] Building CXX object build/network/CMakeFiles/network.dir/WebServer.cpp.o
[build] /home/arch/Github/xbmc-enen/xbmc/xbmc/network/WebServer.cpp:601:13: warning: Value MHD_HTTP_PAYLOAD_TOO_LARGE is deprecated, use MHD_HTTP_CONTENT_TOO_LARGE
[build]   601 |       connectionHandler->errorStatus = MHD_HTTP_PAYLOAD_TOO_LARGE;
[build]       |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                          
[build] [7/13  23% :: 0.028] Building CXX object build/windowing/X11/CMakeFiles/windowing_X11.dir/GLContextEGL.cpp.o
```

Reply:
See tags on https://github.com/Karlson2k/libmicrohttpd/commit/8c644fc1f4d498ea489add8d40a68f5d3e5899fa

Reply:
Perfect, thanks!